### DISCUSSION TITLE
Custom color utility

### DISCUSSION THREAD

**Discussion by federicocappellotto97** [tailwindlabs/tailwindcss#8340] [Help]
Hello, i need a custom utility for every color on my palette.
For example:
```
.class-$color {
   color: $color;
   background-color: $color;
}
```
How should i create it?

**Comment by zslabs** [comment]
https://tailwindcss.com/docs/customizing-colors

See the "Using custom colors" section for instructions.

**Comment by federicocappellotto97** [comment]
I’ve already read it, but i don’t see how to create a custom utility using my color palette
Il 13 mag 2022, 16:10 +0200, Zach Schnackel ***@***.***>, ha scritto:
